src/components/Header.vue

```diff
@@ -9,18 +9,43 @@
       <a href="#">About</a>
     </nav>
     <div class="btns">
-      <ButtonComp class="btn-outline-dark" text="Login" />
-      <ButtonComp class="btn-outline-dark" text="Register" />
+      <button class="btn-outline-dark" @click="openModal" >Login</button>
+      <button class="btn-outline-dark" @click="openRegisterModal">Register</button>
     </div>
   </header>
+  <LoginModal  @close-modal="closeModalHandler" v-if="showModal" />
+  <RegisterModal @close-register-modal="closeRegisterModalHandler" v-if="showRegisterModal" />
 </template>
 
 <script>
-import ButtonComp from "./Button.vue";
+import LoginModal from "./LoginModal.vue";
```

Comment:
use initials like @/components/componentName.vue instead of giving relative path